### DISCUSSION TITLE
ElNotification渲染HTML内容

### DISCUSSION THREAD

**Discussion by Tuise233** [element-plus/element-plus#15845] [Q&A] [answered]
我想使用ElNotification渲染HTML内容，且该内容包含elementplus提供的组件，例如 el-tag，但我测试未显示出正常的样式

**Comment by kooriookami** [comment, marked as answer]
https://cn.vuejs.org/api/built-in-directives.html#v-html
v-html不能解析模板语法

**Reply by lk101** [reply]
我注意到您已经把 #15837 修复了，请问，它的发布计划目前已经有了吗？

**Reply by kooriookami** [reply]
@lk101 今天下午会发一版~
